pom.xml

```diff
@@ -216,6 +217,62 @@
     </build>
 
     <profiles>
+		<profile>
+			<id>rpm</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.codehaus.mojo</groupId>
+						<artifactId>rpm-maven-plugin</artifactId>
+						<version>2.1.5</version>
+						<executions>
+							<execution>
+								<id>generate-rpm</id>
+								<goals>
+									<goal>rpm</goal>
+								</goals>
+							</execution>
+						</executions>
+						<configuration>
+							<group>Applications/Internet</group>
+							<packager>Confluent Packaging</packager>
+							<needarch>false</needarch>
+							<requires>
+								<require>confluent-common</require>
+							</requires>
+							<mappings>
+								<mapping>
+									<directory>/usr/share/java/</directory>
```

Comment:
Is the plugin smart about directories it includes in the RPM. iirc the RPMs (or maybe it was debs...) need to have a single owner. Unless that's just the individual files? Anyway, it'd be good to verify we can use `/usr/share/java` and don't need to use the more specific `/usr/share/java/kafka-connect-jdbc`.